snmp_juniper/README.md

```diff
@@ -0,0 +1,39 @@
+# Agent Check: snmp_juniper
+
+## Overview
+
+Collect SNMP metrics from Juniper appliances, including:
+
+- Juniper EX Ethernet Switches
+- Juniper MX Routers
+- Juniper SRX Firewalls
+
+For details of monitored metrics see the [SNMP integration tile][2]
```

Comment:
nit, but change to

For a complete list of all the metrics collected from Juniper devices, see the [Network Device Monitoring documentation] 

^note I would recommend linking to the NDM docs (https://docs.datadoghq.com/network_performance_monitoring/devices/data), rather than another tile. I think this a little bit more obvious of a pattern to link to documentation

Reply:
This is interesting ! Is this page updated from the `metadata.csv` of the SNMP integration? (cc @pducolin)